packages/containers/src/List/List.container.js

```diff
@@ -203,6 +204,18 @@ class List extends React.Component {
 			}
 		}
 
+		if (props.cellDictionary) {
+			props.list.cellDictionary = Object.keys(props.cellDictionary).reduce((acc, key) => {
+				const current = props.cellDictionary[key];
+				// eslint-disable-next-line no-param-reassign
+				acc[key] = {
+					...omit(current, ['cellRendererComponent', 'cellRendererComponentId']),
```

Comment:
`cellRendererComponentId` is not used here. Why do we have it ?

Reply:
Just forgot it on the next line 🙄